CONTRIBUTORS_GUIDE.md

```diff
@@ -0,0 +1,38 @@
+# Contributor's Guide
+written by @MaxReinisch
+
+
+## Getting Started
+1) Make sure your computer has git and node.js installed.  
+2) Fork the existing project at https://github.com/internetarchive/wayback-machine-chrome/
+3) Navigate to a directory in which you want to clone the project.
+4) `$ git clone https://github.com/<github_account_name>/wayback-machine-chrome.git`
+5) `$ cd wayback-machine-chrome`
+6) `$ npm install`
+7) You should now have a `node_modules` directory.  
+
```

Comment:
How about adding a line addressing how to add upstream as remote
`git remote add upstream https://github.com/internetarchive/wayback-machine-chrome.git`
so that new/first-time contributors following demo contribution (below) don't get stuck at step 2?

Reply:
I also suggest adding a section on syncing your fork or at least providing a link for same such as https://help.github.com/articles/syncing-a-fork/

Reply:
I think the upstream thing will be good for beginners